ui/partials/perf/graphsctrl.html

```diff
@@ -78,6 +78,7 @@
         </p>
         <p ng-hide="tooltipContent.revision">Loading revision...</p>
         <p id="tt-t" class="small" ng-bind="tooltipContent.date"></p>
+        <p id="tt-v" ng-show="tooltipContent.retriggers > 0">Retriggers: {{tooltipContent.retriggers}}</p>
```

Comment:
Could you add a class="small" to this, as we do above? I'd prefer this information be more subtle.